Tasks/DockerV2/task.json

```diff
@@ -135,6 +146,8 @@
     "messages": {
         "AddingNewAuthToExistingConfig": "Adding auth data for registry to Docker config file. Registry: %s.",
         "ConnectingToDockerHost": "DOCKER_HOST variable is set. Docker will try to connect to the Docker host: %s",
+        "CommandNotRecognized": "Command not recognized: %s",
```

Comment:
This seems to be missing from `task.loc.json` and `resources` file. Pls build and commit them.